### ISSUE TITLE
thread 'main' panicked at 'the long_name (second argument) should be longer than a single character

### ISSUE THREAD

**Issue by tofutim** [vasi/qcow2-fuse#1]
```
Tims-Air:qemu_vms tim$ qcow2-fuse raspbian-stretch.qcow mnt
thread 'main' panicked at 'the long_name (second argument) should be longer than a single character, or an empty string for none', /Users/tim/.cargo/registry/src/github.com-1ecc6299db9ec823/getopts-0.2.17/src/lib.rs:572:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```
Any idea how I can make this work?

**Comment by tofutim** [comment]
> Only version 3 (aka "qcow2 1.1") is supported; version 2 is not supported.

Oops I didn't read this, perhaps that is the problem? I used qemu-img version 2.11.1 to create the qcow.

**Comment by ybart** [comment]
It should have been fixed by this:

https://github.com/vasi/qcow2-fuse/pull/3

Could you retry to see if that fixed it for you?

**Comment by hiburn8** [comment]
I get the same issue with a 1.1 file that i have confirmed is not corrupt. is OSX support broken? I'm not sure the example command works.

**Comment by dirkbolte** [comment]
I can into the same issue and it already fails with `qcow2-fuse -h` . The fix of #3 completely works for me - but it isn't published to cargo yet. So every `cargo install qcow2-fuse` fails atm as these are built against getopts 0.2.21 (same semver patch version, thus the newest is fetched).

**Comment by vasi** [comment]
ok, published a new version, thanks!
